Sources/Rendering/OpenGL/HardwareSelector/index.js

```diff
@@ -44,6 +45,7 @@ function vtkOpenGLHardwareSelector(publicAPI, model) {
         model.framebuffer.create(size[0], size[1]);
         model.framebuffer.populateFramebuffer();
       }
+      model.framebuffer.bind();
```

Comment:
shouldn't you call bind only in `else()` because `populateFramebuffer()` is called line 45/46 and your comment line 38 says `populateFramebuffer()` already calls `bind()` ?

Reply:
Good point, I'll put it in the else block. I assum `bind` is idempotent, but best to not make assumptions here. :)